install/create_system_config_cds.md

```diff
@@ -0,0 +1,394 @@
+# Create system configuration using cluster discovery service
+
+This stage walks the user through creating the configuration payload for the system using Cluster Discovery Service (CDS) to generate seed files and paddle file.
+
+Run the following steps before starting any of the system configuration procedures.
+
+1. (`pit#`) Make the `prep` and `cds` directory.
+
+   ```bash
+   mkdir -pv "${PITDATA}/prep/cds"
+   ```
+
+1. (`pit#`) Change into the `cds` directory.
+
+   ```bash
+   cd "${PITDATA}/prep/cds"
+   ```
+
+1. (`pit#`) Set `CVT_PATH` variable.
+
+   ```bash
+   CVT_PATH=/opt/src/cluster-config-verification-tool/
+   ```
+
+## Topics
+
+1. [Customize `system_config.yaml`](#1-customize-system_configyaml)
+1. [Configure the management network](#2-configure-the-management-network)
+1. [Configure auto-discovery](#3-configure-auto-discovery)
+    1. [Configure `dnsmasq`](#31-configure-dnsmasq)
+    1. [Verify and reset the BMCs](#32-verify-and-reset-the-bmcs)
+    1. [Verify the discover status](#33-verify-the-discover-status)
+    1. [Collect and compare the inventory data](#34-collect-and-compare-the-inventory-data)
+1. [Generate seed files and paddle file](#4-generate-seed-files-and-paddle-file)
+1. [Stop discovery](#5-stop-discovery)
+1. [Run CSI](#6-run-csi)
+1. [Prepare Site Init](#7-prepare-site-init)
+1. [Initialize the LiveCD](#8-initialize-the-livecd)
+1. [Next topic](#next-topic)
+
+## 1. Customize `system_config.yaml`
+
+1. (`pit#`) Change into the `prep` directory.
+
+   ```bash
+   cd "${PITDATA}/prep"
```

Comment:
This fenced code block is opened but not closed, which is likely causing many of the linter errors.

````suggestion
   cd "${PITDATA}/prep"
   ```
````